frontend/src/constants.ts

```diff
@@ -76,6 +79,13 @@ export const EmptyConstants = new Constants({
   NAUTILUS_PROJECT_QUOTA: 104857600
 })
 
+// We need to use 2 as the base instead of the default base.
+// if we use the default base, we will have the file rejected for less than the limit
+// We use jedec, rather than the default iec to make the file size display more readable.
+// After using jedec, the file will display MB instead of MiB
 export const humanifyFileSize = partial({ base: 2, standard: 'jedec', output: 'string' })
```

Comment:
```suggestion
export const humanifyFileSize = partial({ base: 10, standard: 'jedec', output: 'string' })
```

I personally prefer `iec` but jedec is what's used by operating systems (at least macOS) and we should thus stick to it. We need to make sure we configure the backend with `100MB` then and not `100MiB` to prevent confusion… and align environ values (`100000000` for instance).

Reply:
> We should stick with `jedec`, but we can use 10 as the base.

That's what I said